app/src/main/java/com/sacramento/movies/ui/movies/MoviesFragment.kt

```diff
@@ -122,4 +126,14 @@ class MoviesFragment : Fragment() {
 
     }
 
+    private fun refreshList() {
```

Comment:
This method doesn't refresh the list. Name is confusing
The same comment for SearchMovieByNameFragment